Python/ds/Tree_postorder_iterative.py

```diff
@@ -0,0 +1,56 @@
+# Python program for iterative postorder traversal
+# A binary tree node 
+class Node: 
+	
+	# Constructor to create a new node 
+	def __init__(self, data): 
+		self.data = data 
+		self.left = None
+		self.right = None
+
+# An iterative function to do postorder traversal of a given binary tree 
+def postorderIterative(root): 
+
+	if root is None: 
+		return		
+	
+	s1 = [] # Create two stacks 
+	s2 = [] 
+	
+	s1.append(root) # Push root to first stack 
+	
+	while s1: 		# Run while first stack is not empty 
+		node = s1.pop() # Pop an item from s1 and 
+		s2.append(node) # append it to s2 
+	
+		# Push left and right children of removed item to s1 
+		if node.left: 
+			s1.append(node.left) 
+		if node.right: 
+			s1.append(node.right) 
+
+		# Printing the result
+	while s2: 
+		node = s2.pop() 
+		print node.data, 
+
+# Driver program to test above function 
+root = Node(10) 
```

Comment:
Hey @rutujadhanawade I can see the hardcoded value here, can you please make it for general input.